client/src/components/reviews/Reviews.jsx

```diff
@@ -34,13 +34,39 @@ class Reviews extends React.Component {
     });
   }
 
+  sortList (e) {
+    let sorted;
+    if (e.target.value === 'Helpful') {
+      sorted = this.state.reviewList.sort((a, b) => {
+        return b.helpfulness - a.helpfulness;
+      });
+    }
+    if (e.target.value === 'Newest') {
+      sorted = this.state.reviewList.sort((a, b) => {
+        return new Date(b.date) - new Date(a.date);
+      });
+    }
+    if (e.target.value === 'Relevance') {
+      sorted = this.state.reviewList.sort((a, b) => {
+        if (a.helpfulness === b.helpfulness) {
+          return new Date(b.date) - new Date(a.date);
+        }
+        return b.helpfulness - a.helpfulness;
+      });
+    }
+    console.log('sort,', sorted);
+    this.setState({
+      reviewList: sorted
```

Comment:
Consider storing the sorted list in a separate state? This would allow you to save your initial list load that is not sorted if the user wants to revert back and reduces sorting calculation if they switch between sort options?